src/auth/src/component.cairo

```diff
@@ -129,6 +108,38 @@ use core::result::ResultTrait;
         }
     }
 
+
+    #[embeddable_as(WebauthnCamel)]
+    impl WebauthnImplCamel<
```

Comment:
Do we actually need the camel one? Is there some existing contracts already deployed relying on this?
If not, I am supportive of having only one, standard, snake_case.

Reply:
It's supposed to be analogous to the session implementation (https://github.com/neotheprogramist/cairo-webauthn/blob/f03fd6d7967d23f569954286241383cc441ad978/src/session/src/interface.cairo).
I've heard that the camel case will be deprecated soon, are you familiar with a more precise timeline?
I will of course remove the camel case.

Reply:
My understanding was that camel-case is not yet deprecated. It's supposed to be analogous to the session implementation (https://github.com/neotheprogramist/cairo-webauthn/blob/f03fd6d7967d23f569954286241383cc441ad978/src/session/src/interface.cairo). 
We're not constrained by any deployed contracts since there are none :)). 
Since you recommend removing it I will gladly do that. Should I also remove the camel interface in session?
Also, do you know when will the camel case be officially deprecated?

Reply:
Removed.

Reply:
Thanks for the feedback @ThePinion!

> Also, do you know when will the camel case be officially deprecated?

Officially, it will always be possible to create camel case functions I think. I'm not sure that cairo compiler will enforce snake case.
However, the transition to cairo 1 + the interfaces being standardized with the `SRC5`, snake case if being the one used here.

So yes, if we can stick to snake case everywhere it could be great for consistency and starknet standardization. :+1: